app/models/like.rb

```diff
@@ -0,0 +1,8 @@
+class Like < ApplicationRecord
+  belongs_to :users
+  belongs_to :posts
+
```

Comment:
### [OPTIONAL]
**Hint** : Just like we have in posts, we can do the same here `after_save :update_likes_counter`